### DISCUSSION TITLE
Is it possible to rename the indirect expand in result json?

### DISCUSSION THREAD

**Discussion by benny856694** [pocketbase/pocketbase#1809] [Q&A] [answered]
I just think the naming like "comments(post) " doesn't look pretty

**Comment by ganigeorgiev** [comment, marked as answer]
No, it is not possible at the moment.

In my opinion, it will add unnecessary complication to the already complex enough expand syntax. Additionally, allowing to provide an alias for the key could result in expand keys collision with unexpected behavior (we'll have to either throw an error or silently overwrite the key).
